command/s3_put.go

```diff
@@ -309,6 +315,10 @@ retryLoop:
 
 				fpath = filepath.Join(filepath.Join(s3pc.workDir, s3pc.LocalFilesIncludeFilterPrefix), fpath)
 				err = s3pc.bucket.Upload(ctx, remoteName, fpath)
+
+				if s3pc.Visibility == "signed" {
+					s3pc.FileKey = remoteName
```

Comment:
In the case of a multi-file upload, it looks like since we're updating the FileKey for each file in filesList the value of the FileKey that we use in `attachFiles` will be the same for all the files.

Reply:
I changed it to instead use remoteFileName in attachFiles as the fileKey. Thanks for pointing it out!